BERT Sentiment Analysis/test.py

```diff
@@ -10,6 +10,7 @@
 tokenizer = BertTokenizer.from_pretrained(PRE_TRAINED_MODEL_NAME)
 
 class SentimentClassifier(nn.Module):
+    print("module")
```

Comment:
**Issue**: Incorrect indentation after class definition

**Reason**: In Python, the first line inside a class must be indented with exactly 4 spaces. The 'print("module")' statement is incorrectly placed and will cause a syntax error.

**Suggestion**: Remove or properly indent the 'print("module")' statement to align with the rest of the class methods.